### PR TITLE
[Docs] Add solution for markdown-lint installation issue

### PR DESCRIPTION
**Pull request by dreamerlin**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -49,6 +49,21 @@ From the repository folder
...
 ```
 
+If you are facing an issue when installing markdown lint, you may install ruby for markdown lint by the following stpes
+
+```shell
+# install rvm
+curl -L https://get.rvm.io | bash -s -- --autolibs=read-fail
+
+# set up environment
+echo 'source $HOME/.bash_profile' >> ~/.bashrc
```
Comment (innerlee) [inline .github/CONTRIBUTING.md:59]: this line is not friendly

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -49,6 +49,21 @@ From the repository folder
...
+source ~/.profile
```
Comment (innerlee) [inline .github/CONTRIBUTING.md:60]: why?

**Comment by dreamerlin** [comment]
These commands are copied from mmdet. How about help to add script to install ruby>2.5 in `setup` repo ? Could you help it @innerlee

**Comment by dreamerlin** [comment]
> These commands are copied from mmdet. How about help to add script to install ruby>2.5 in `setup` repo ? Could you help it @innerlee

ok i found it

**Comment by innerlee** [comment]
btw the setup repo lacks some steps of post-installation. There are several env variables need to be defined. Do you have intersest to have a try, and find out the missing steps?

**Comment by dreamerlin** [comment]
> btw the setup repo lacks some steps of post-installation. There are several env variables need to be defined. Do you have intersest to have a try, and find out the missing steps?

I am afraid I have no time these days. @congee524 , Could you help to do it?

**Comment by congee524** [comment]
> > btw the setup repo lacks some steps of post-installation. There are several env variables need to be defined. Do you have intersest to have a try, and find out the missing steps?
> 
> I am afraid I have no time these days. @congee524 , Could you help to do it?

okk

**Comment by congee524** [comment]
> btw the setup repo lacks some steps of post-installation. There are several env variables need to be defined. Do you have intersest to have a try, and find out the missing steps?

Do you mean the missing steps in the process of installing ruby, or all the install scrips?

**Review by innerlee — APPROVED** [review]

**Comment by innerlee** [comment]
Sorry clicked the wrong button, didn't meant to approve this. My comments are still valid.

**Review by innerlee — APPROVED** [review]

**Pull request merged**